core/domain/collection_services.py

```diff
@@ -313,12 +411,19 @@ def get_next_exploration_ids_to_complete_by_user(user_id, collection_id):
     """Returns a list of exploration IDs in the specified collection that the
     given user has not yet attempted and has the prerequisite skills to play.
 
-    Returns the collection's initial explorations if the user has yet to
-    complete any explorations within the collection. Returns an empty list if
-    the user has completed all of the explorations within the collection.
+    Args:
+        user_id: str. User id of a given user.
+        collection_id: str. Collection id of a given collection.
 
-    See collection_domain.Collection.get_next_exploration_ids for more
-    information.
+    Returns:
+        A list of exploration IDs in the specified collection that the given
+        user has not attempted and has the prerequisite skills to play. Returns
```

Comment:
Hm -- has not attempted, or has not completed? The former option contradicts the next sentence.